supervisor/api/ingress.py

```diff
@@ -42,6 +42,11 @@
 class APIIngress(CoreSysAttributes):
     """Ingress view to handle add-on webui routing."""
 
+    def __init__(self, coresys: CoreSys) -> None:
```

Comment:
Let's not change the style of our API views. It will work also if we not set coresys over init